spec/index.html

```diff
@@ -740,28 +740,25 @@ <h3>Literals</h3>
       <li>If the literal is a <a>language-tagged string</a>,
         then the literal value is a pair consisting of its <a>lexical form</a>
         and its <a>language tag</a>, in that order.</li>
-      <li>if the literal is a <a>directional language-tagged string</a>, then the literal value is
+      <li>If the literal is a <a>directional language-tagged string</a>, then the literal value is
         a tuple of its <a>lexical form</a>, its <a>language tag</a>, and its <a>base direction</a>,
         likewise in that order.</li>
-
-      <li>If the literal's <a>datatype IRI</a> is in the set of
-        <a>recognized datatype IRIs</a>, let <var>d</var> be the
-        <a>referent</a> of the datatype IRI.
+      <li>If the literal's <a>datatype IRI</a> is handled by an RDF processor,
         <ul>
-          <li>If the literal's <a>lexical form</a> is in the <a>lexical space</a>
-            of <var>d</var>, then the literal value is the result of applying
-            the <a>lexical-to-value mapping</a> of <var>d</var> to the
+          <li>if the literal's <a>lexical form</a> is in the <a>lexical space</a>
+            of datatype, then the literal value is the result of applying
+            the <a>lexical-to-value mapping</a> of the datatype to the
             <a>lexical form</a>.</li>
```

Comment:
```suggestion
            <a>lexical form</a>;</li>
```

Reply:
For reasons unknown, the most (not all) bulleted lists in the doc have `'` on each list item. It would be better to deal with this separately because this PR is growing in scope already and I think keeping it as focused as possible will help get it accepted.

Reply:
https://github.com/w3c/rdf-concepts/issues/127